src/main/java/com/johnstarich/moviematcher/controllers/JsonService.java

```diff
@@ -36,6 +36,9 @@ public final void init() {
 		Filter jsonAttribute = (req, resp) -> {
 			resp.type("application/json");
 			Map jsonMap = null;
+			System.out.println("Body:");
+			System.out.println(req.body());
+			System.out.println(req.body());
```

Comment:
Sorry, this is a work in progress! I will be removing those println statements as soon as I can figure out what went wrong in production with login. :smile: